controllers/admins.js

```diff
@@ -0,0 +1,40 @@
+const fs = require('fs');
+const admins = require('../data/admin.json');
+
+//Get all admins
+
+const getAdmins = (req, res) => {
+    res.status(200).json(
+        res.json(admins)
+    )
+}
+
+//Update an admin
+
+const updateAdmin = (req, res) => {
+    const found = admins.some(admin => admin.id === parseInt(req.params.id));
+
+    if (found){
+        admins.forEach(admin => {
+            if (admin.id === parseInt(req.params.id)){
+                !req.query.email ? null : admin.email = req.query.email;
+                !req.query.password ? null : admin.password = req.query.password
+            }   
+        })
+    }   else{
+        res.status(400).json({msg: `Admin with id: ${req.params.id} was not found`})
+    }
```

Comment:
if you use a `map` to find the admin and update it you are iterating all the list just once. If you use a `some` and then a `forEach` you are iterating the list twice. In performance terms, the second option is more expensive.